include/galsim/hsm/PSFCorr.h

```diff
@@ -137,6 +137,11 @@ namespace hsm {
      *                           exception.
      * @param ksb_moments_max    Use moments up to ksb_moments_max order for KSB method of PSF
      *                           correction.
+     * @param ksb_sig_weight     The width of the weight function for the KSB method.  Normally,
+     *                           this is computed from the measured moments of the galaxy image.
+     *                           This keyword overrides this calculation.
+     * @param ksb_sig_factor     Factor by which to multiply the weight function width for the KSB
```

Comment:
One might wonder if you are supposed to use only one of these (but not both).  the code is set up so that using both is an option; might mention this explicitly?